lib/dotenv_validator.rb

```diff
@@ -133,7 +138,7 @@ def self.sample_file
   #
   # Taken from Dotenv source code.
   def self.root
-    root_or_pwd = Pathname.new(ENV["RAILS_ROOT"] || Dir.pwd)
+    root_or_pwd = Pathname.new(ENV['RAILS_ROOT'] || Dir.pwd)
```

Comment:
Why the change to single quotes is needed?

Reply:
@bronzdoc If I did not change that, rubocop would complain.